discovery/README.md

```diff
@@ -0,0 +1,56 @@
+# Discovery of PFIs
+
+Paritipating Financial Institutions (PFIs) will likely wish to make their presence known to the public.
+
+This part of the protocol shows how this can be done in a decentralized fashion. 
+
+## Service Endpoints
+
+For a PFI to be discoverable, it must publish a DID Document with a service endpoint of type "PFI" at a minimum (from serviceEndpoints section of DID Document):
+
+```json
+      {
+        id:'pfi',
+        type: 'PFI',
+        serviceEndpoint: 'https://pfi.mic4.com',
+      }
+```      
+
+### Optional currency pair declaration
+
+To help clients find the right PFI for their needs, a PFI may also declare what currency pairs they are able to offer:
+
+```json
+      {
+        id:'pfi',
+        'type': 'PFI',
+        'serviceEndpoint': 'https://pfi.mic4.com AUD-USD,AUD-TZN,USD-BTC',
```

Comment:
hmm, is this a convention we're proposing here that wallets have to be aware of, in order to crawl the DID document whose serviceEndpoint type is PFI and hint at available pairs? i thought that fetching offering  might do a better job of this.

Reply:
fetching offering does do a better job, this is a "hint" - say there are 100s returned, then it would have to fire off 100s of requests. Maybe that is ok? if there were 1000, maybe a heavy load but I guess that is a "good problem to have"? So perhaps this is over thinking it. The offers request is still the difinitive thing, so perhaps this is confusing?

Reply:
@jiyoontbd yeah I think perhaps can remove it, keep the endpoint as a simple endpoint (be that http or whatever).